### ISSUE TITLE
Run a link-checker on new website

### ISSUE THREAD

**Issue by Axel-Naumann** [root-project/web#173]

**Comment by eguiraud** [comment]
(would be nice to have it running as a github action)

**Comment by stwunsch** [comment]
I'm currently running a linkchecker on the website (we have lots of dead links :/), however, this should be definitely an action: https://github.com/marketplace?type=actions&query=link+checker

**Comment by stwunsch** [comment]
[linkchecker.log](https://github.com/root-project/web/files/4790828/linkchecker.log)

We have massive failures in the release notes, e.g., because we point to savannah. I'll rerun with proper regex filters to get a meaningful report. However, we have around 5000 links on the website :D

**Comment by eguiraud** [comment]
nice! about savannah, that's probably the release notes for the v5 series, i don't think we care too much

**Comment by stwunsch** [comment]
[linkchecker_nosavannah.log](https://github.com/root-project/web/files/4790941/linkchecker_nosavannah.log)

And without savannah, still massive issues with the release notes (dead links to ancient releases, ...). I'll make another version without the release notes at all so we have a baseline to fix the important ones.

**Comment by stwunsch** [comment]
[linkchecker_noreleasenotes.log](https://github.com/root-project/web/files/4791080/linkchecker_noreleasenotes.log)

Down to 61 errors :) I'll look at these.

```
That's it. 3743 links in 3956 URLs checked. 1 warning found. 61 errors found.
```

**Comment by stwunsch** [comment]
Here is a first batch of fixes: https://github.com/root-project/web/pull/195

**Comment by stwunsch** [comment]
The action is here: https://github.com/root-project/web/pull/196

It's running for PRs and commits to `gh-pages`.

**Comment by stwunsch** [comment]
Fixed by #195, #206, #211 and #212
